pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -201,5 +201,26 @@ in buildPythonPackage {
     license = licenses.asl20;
     maintainers = with maintainers; [ jyp abbradar cdepillabout ];
     platforms = [ "x86_64-linux" "x86_64-darwin" ];
+    knownVulnerabilities = [
```

Comment:
FWIW, I think this better be `optionals (versionOlder version "2.12.0") [ ... ]` in case of the `-bin` package (there's a bunch of wheels listed in `binary-hashes.nix`)